### ISSUE TITLE
With a word or symbol selected, how do I extend selection to include more and more words?

### ISSUE THREAD

**Issue by RKBK** [meow-edit/meow#358]
If I am working in a sentence, I might want to select three words. Then, I would probably do `avy-jump-to-char` (write characters to match to word in buffer and jump to a word that matches) to get to the first word. Then I'd press w to select the word. But, how do I then most easily extend my selection to include the next word? Selecting full lines with `x` lets you press x multiple times to select more and more lines, but `w` doesn't work like that. Also, pressing `]` to extend selection to e.g. the full line is not what I want, because I might just want to select three words. I guess I can do `shift-<right arrow>` to extend the region to the next word, but then my intuition is to press `w` to select _also_ that word, but that actually changes the region to include _only_ that next word.

What's the recommended way to achieve this in meow?

**Comment by DogLooksGood** [comment]
I gave an example in your another question. And there are some material about the concept behind the command design in meow.

https://github.com/meow-edit/meow/blob/master/EXPLANATION.org#selection

**Comment by RKBK** [comment]
I think part of the reason I got confused here, even though I had worked through the tutorial including the part about extending selection, is that I am doing a lot of my typing and editing in org-mode. The extension functionality (at least the part which shows hints for extending selection) is turned off by default, as I learned in #110 . I'm using a fixed-width font, so I've now enabled the functionality also in org-mode. Extending selection is no problem now!
